vms/example/xsvm/cmd/issue/transfer/cmd.go

```diff
@@ -51,19 +65,23 @@ func transferFunc(c *cobra.Command, args []string) error {
 	}
 	stx, err := tx.Sign(utx, config.PrivateKey)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
-	txJSON, err := json.MarshalIndent(stx, "", "  ")
+	issueTxStartTime := time.Now()
+	txID, err := client.IssueTx(ctx, stx)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
-	issueTxStartTime := time.Now()
-	txID, err := client.IssueTx(ctx, stx)
+	err = client.WaitForAcceptance(ctx, address, nonce)
 	if err != nil {
-		return err
+		return nil, err
 	}
-	log.Printf("issued tx %s in %s\n%s\n", txID, time.Since(issueTxStartTime), string(txJSON))
-	return nil
+
+	return &tx.TxIssueStatus{
+		Tx:        stx,
+		TxID:      txID,
+		StartTime: issueTxStartTime,
```

Comment:
q: do we plan to use this time for some kind of metrics? This time is related to node clock (not any time of network agreed time) and I wonder if it should be here instead of the e2e test code